pkg/apply/merge.go

```diff
@@ -209,6 +209,10 @@ func mergeLabels(current, updated *uns.Unstructured) {
 	}
 }
 
+func removeAddressPool(slice []metallbv1alpha.AddressPoolSpec, index int) []metallbv1alpha.AddressPoolSpec {
+	return append(slice[:index], slice[index+1:]...)
```

Comment:
Couldn't this panic?

Reply:
Assuming index is at most len(slice), it won't. I just tested it because I was not sure https://play.golang.org/p/sJCHrlg-QZB

Reply:
it won't I tested too here https://play.golang.org/p/1P1IrbIJYEF, I tried boundries and it seems to work fine